fvm/blueprints/scripts/systemChunkTransactionTemplate.cdc

```diff
@@ -11,5 +12,11 @@ transaction {
             from: NodeVersionBeacon.HeartbeatStoragePath)
                 ?? panic("Couldn't borrow NodeVersionBeacon.Heartbeat Resource")
         versionBeaconHeartbeat.heartbeat()
+
+        let randomBeaconHistoryHeartbeat = serviceAccount.borrow<&RandomBeaconHistory.Heartbeat>(
+            from: RandomBeaconHistory.HeartbeatStoragePath)
+                ?? panic("Couldn't borrow RandomBeaconHistory.Heartbeat Resource")
+        let sor = randomSourceHistory()
```

Comment:
is `randomSourceHistory` a global function?

Reply:
Are users going to use the `randomSourceHistory`? If it's only injected by FVM, it is not going to be available in any tooling. We might want to consider moving this to the Cadence stdlib?

Reply:
> Are users going to use the randomSourceHistory?

No. The idea behind this is that `randomSourceHistory` is available only in the system transaction and only the system transaction needs to know about it. This is then used to fill the `RandomBeaconHistory` contract which can then be just a normal contract without any special elements or tooling.

Reply:
I see, that makes sense, thanks for the explanation! Could you please add this as a comment to the code?